main/src/ca/uwaterloo/flix/MutationTester.scala

```diff
@@ -0,0 +1,86 @@
+package ca.uwaterloo.flix
+
+import ca.uwaterloo.flix.api.Flix
+import ca.uwaterloo.flix.language.ast.Ast.Constant
+import ca.uwaterloo.flix.language.ast.{Ast, TypedAst}
+import ca.uwaterloo.flix.language.ast.TypedAst.Expr
+import ca.uwaterloo.flix.util._
+
+import java.io.File
+
+object MutationTester {
+    def run(files: Seq[File], options: Options, tester: String, testee: String): Unit = {
+        val flix = new Flix
+        flix.setOptions(options)
+        flix.addSourceCode("testee",
+            s"""
+            |def main(): Unit \\ IO =
+            |    println(Test.one())
+            |
+            |mod Test {
+            |    pub def one(): Int32 = 1
+            |}""".stripMargin)
+
+        flix.addSourceCode("tester",
+            s"""
+            |mod MainTest {
+            |    @test
+            |    def testConstant(): Bool =
+            |        Assert.eq(Test.one(), 1)
+            |}""".stripMargin)
+        val root = flix.check().unsafeGet
+        val root1 = mutate(root, testee)
+        val res = flix.codeGen(root1).unsafeGet.getTests
+        println(res)
```

Comment:
You will want to run the tests here. You can look at `Tester` to see how that is done. I suggest not to use `Tester` but to roll your own code.